background_test.go

```
@@ -269,3 +273,136 @@ func TestProcessAutoTagDocuments(t *testing.T) {
+					if tag == pdfOCRCompleteTag {
+						hasCompleteTag = true
+						break
+					}
+				}
+
+				env.setMockResponse(updatePath, func(w http.ResponseWriter, r *http.Request) {
+					if hasCompleteTag {
+						// For documents with complete tag, verify only autoOcrTag is removed
+						var updateRequest map[string]interface{}
+						body, _ := io.ReadAll(r.Body)
+						json.Unmarshal(body, &updateRequest)
```

golangci-lint (1.64.8)

Error return value of `json.Unmarshal` is not checked

(errcheck)